packages/primitives/src/ListItem/ListItem.stories.tsx

```diff
@@ -74,10 +74,14 @@ export const NonInteractive: StoryFn<typeof ListItemRoot> = (args) => (
 
 export const WithDescription: StoryFn<typeof ListItemRoot> = (args) => (
   <ListItemRoot {...args}>
-    <ListItemImage src="https://api.staging.ndla.no/image-api/raw/Ide.jpg" alt="En lyspære" height={40} />
+    <ListItemImage
+      src="https://api.staging.ndla.no/image-api/raw/Ide.jpg"
+      alt="En lyspære"
+      css={{ minHeight: "40px", maxHeight: "72px", minWidth: "56px", maxWidth: "100px" }}
+    />
     <ListItemContent>
       <styled.div css={{ display: "flex", flexDirection: "column", gap: "4xsmall" }}>
-        <ListItemHeading asChild consumeCss>
+        <ListItemHeading css={{ alignSelf: "flex-start" }} asChild consumeCss>
```

Comment:
Er ikkje dette berre en oppheving av det du gjør i recipe for title?

Reply:
For å få midtstilt tittel i liste måtte jeg først legge til tittel på listItem, og det påvirket både List story og With Description Story. Jeg kan sikkert legge til flere variabler i ListItem, men føler det kanskje blir litt uoversiktelig?
Alternativt kan jeg også legge til en ny export av typen"ListItemHeadingWithDescription"?
Hedvig mente også det ville være svært sjelden vi fikk bruk for beskrivelse i denne sammenheng, så jeg vet ikke hvor avansert denne bør være.